src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.ts

```diff
@@ -199,7 +199,15 @@ export class ExamParticipationCoverComponent implements OnInit, OnDestroy {
         if (this.testRun) {
             return this.nameIsCorrect && this.confirmed && !!this.exam;
         }
-        return !!(this.nameIsCorrect && this.confirmed && this.exam && this.exam.visibleDate && this.exam.visibleDate.isBefore(this.serverDateService.now()));
+        const now = this.serverDateService.now();
+        return !!(
+            this.nameIsCorrect &&
+            this.confirmed &&
+            this.exam &&
+            this.exam.visibleDate &&
+            this.exam.visibleDate.isBefore(now) &&
+            now.add(5, 'minute').isAfter(this.exam.startDate!)
```

Comment:
Please use a constant here as well and use a more compile safe variant where you don't need to use a string

Reply:
I added the constant. But I am not sure what to do to remove the string usage, apart from also using a string constant maybe? The method expects a string that maps to 'minutes' | 'minute' | 'm' (documentation) and afaik this is the only method we can use in this context. Would be great to get some feedback on that :)

Reply:
I added the constant. But I am not sure what to do to remove the string usage, apart from also using a string constant maybe? The method expects a string that maps to 'minutes' | 'minute' | 'm' (documentation) and afaik this is the only method we can use in this context. Would be great to get some feedback on that :)

Reply:
I think using `.add(number, string)` with seconds, minutes, hours, days, months etc as string is the standard way to manipulate moments (https://momentjs.com/docs/#/manipulating/add/), wee use this all the time. We normally do use the plural, though. (-> `now.add(5, 'minutes')` but minute can be used, too, as well as just 'm')